### PR TITLE
types

### PR DESCRIPTION
**Pull request by NicoZweifel**
## Summary
adds types
## Test Plan
`yarn prebuild` worked

### CODE REVIEW

**Review by tobias-mintlify — APPROVED** [review]

**Comment by tobias-mintlify** [comment]
Maybe we should merge this at the same time? https://github.com/mintlify/mint/pull/475#pullrequestreview-1337081833

File: client/prebuild/createPage.ts [hunk 1]
```diff
@@ -61,32 +62,37 @@ const preParseMdxHelper = async (fileContent: Compatible, contentDirectoryPath:
...
         if (node == null) {
           return;
         }
-        if (node.name === 'img' || node.name === 'source') {
-          const srcAttrIndex = node.attributes.findIndex(
+        const n = node as Node & {
+          name: string;
+          url: string;
+          attributes: { name: string; value: string }[];
+        };
```
Comment (hahnbeelee) [inline client/prebuild/createPage.ts:74]: The types for remark are so generic and hard to work with 😢

Comment (NicoZweifel) [reply client/prebuild/createPage.ts:74]: Yea, I didn't find another way of asserting the type than that unfortunately.

**Comment by NicoZweifel** [comment]
> Maybe we should merge this at the same time? https://github.com/mintlify/mint/pull/475#pullrequestreview-1337081833

It's targeting this branch, I'll close it so we can move on there.

**Pull request merged**